next/components/forms/simple-components/BannerBasic.tsx

```diff
@@ -0,0 +1,124 @@
+import ArrowRightIcon from '@assets/images/forms/arrow-right.svg'
+import cx from 'classnames'
+import AccountMarkdown from 'components/forms/segments/AccountMarkdown/AccountMarkdown'
+import Image from 'next/legacy/image'
+
+import Button from './Button'
+
+type BannerBasicProps = {
+  header: string
+  content?: string | null
+  imagePath?: string
+  buttonPrimaryText?: string
+  onPressPrimary?: () => void
+  buttonSecondaryText?: string
+  onPressSecondary?: () => void
+  buttonTertiaryText?: string
+  linkTertiary?: string
+  textPosition?: 'left' | 'right'
+  className?: string
+}
+
+const BannerBasic = ({
+  header,
+  content,
+  imagePath = '',
+  buttonPrimaryText,
+  onPressPrimary,
+  buttonSecondaryText,
+  onPressSecondary,
+  buttonTertiaryText,
+  linkTertiary,
+  textPosition = 'left',
+  className,
+}: BannerBasicProps) => {
+  const imageElement = (
```

Comment:
This get recreated everytime component rerenders, I would just use `flex-row` or `flex-row-reverse` on the parent component accordingly.

Reply:
I agree with Marek. Even tho it may be easier to reed it in browser inspector, it's less readable in code and produces potential issues.
I definitely vote for `flex-row` and `flex-row-reverse`

Reply:
Used `flex-row` and `flex-row-reverse`